angularSTEP/src/app/item-dialog/item-dialog.component.html

```diff
@@ -0,0 +1,32 @@
+<div *ngIf="add"> 
+  <h1 mat-dialog-title>Add Item to Shopping List</h1>
+<!-- <div mat-dialog-content>
```

Comment:
Is this commented code expected to be replaced in the near future or should it be deleted now?

Reply:
I'm deleting it now. It escaped my notice when I went through and cleaned up my code.